pkg/codegen/internal/test/testdata/kubernetes-pod.pp.py

```diff
@@ -15,7 +15,7 @@
             "resources": {
                 "limits": {
                     "memory": "20Mi",
-                    "cpu": 0.2,
+                    "cpu": "0.2",
```

Comment:
Is this a regression? Shouldn't it remain `0.2` rather than `"0.2"`?

Reply:
No. This should be a string. 
https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu

And if you see the [Go output](https://github.com/pulumi/pulumi/blob/master/pkg/codegen/internal/test/testdata/kubernetes-pod.pp.go#L26) or the [Dotnet output](https://github.com/pulumi/pulumi/blob/master/pkg/codegen/internal/test/testdata/kubernetes-pod.pp.cs#L30) it's a string. I'm not sure why it wasn't caught before, but this is the correct translation.

Reply:
FWIW in go the type of `limits` is a `pulumi.StringMap`:

```go
// Limits describes the maximum amount of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/
	Limits pulumi.StringMapInput `pulumi:"limits"`
```

Reply:
cc @lblackstone this is still a number in the [TS translation](https://github.com/pulumi/pulumi/blob/master/pkg/codegen/internal/test/testdata/kubernetes-pod.pp.ts#L18), should probably always be a string?